### PR TITLE
modulesync 0.12.2 + Improvements

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
I'm trying to yak shave the tests to get more details about:
* https://github.com/voxpupuli/puppet-zabbix/issues/267
* https://github.com/voxpupuli/puppet-zabbix/issues/266
* https://github.com/voxpupuli/puppet-zabbix/issues/265

**Comment by daenney** [comment]
This is too many changes in one go, which makes it really hard to follow in a PR. I would suggest you split out the improvement and the modulesync and raise them as PRs in that order.

**Comment by jyaworski** [comment]
This is good. The improvement is very minor, then rubocop stuff.
